solvers/glmnet.py

```diff
@@ -38,18 +41,19 @@ def set_objective(self, X, y, lmbd, fit_intercept):
         self.X, self.y, self.lmbd = X, y, lmbd
         self.fit_intercept = fit_intercept
 
-        self.lmbd_max = np.max(np.abs(X.T @ y))
         self.glmnet = robjects.r['glmnet']
 
-    def run(self, n_iter):
-        fit_dict = {"lambda.min.ratio": self.lmbd / self.lmbd_max}
+    def run(self, tol):
+        maxit = 0 if tol == INFINITY else 1_000_000
```

Comment:
@jolars I did this following your remark, but it did not result in a common objective value, and I don't know why...